src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/PlacementRequestBookingSummaryTransformer.kt

```diff
@@ -1,19 +1,32 @@
 package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
 
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.BookingSummary
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BookingEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas1SpaceBookingEntity
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.PlacementRequestEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas1.Cas1SpaceBookingSummaryTransformer
 
 class PlacementRequestBookingSummaryTransformer(
   private val bookingSummaryTransformer: BookingSummaryTransformer,
+  private val cas1SpaceBookingSummaryTransformer: Cas1SpaceBookingSummaryTransformer,
 ) {
 
   fun getBookingSummary(placementRequest: PlacementRequestEntity): BookingSummary? {
-    val booking = placementRequest.booking
+    return placementRequest.booking?.let {
+      getBookingSummary(placementRequest.booking!!)
+    } ?: getSpaceBookingSummary(placementRequest.spaceBookings)
+  }
 
-    if (booking != null && !booking.isCancelled) {
+  private fun getBookingSummary(booking: BookingEntity): BookingSummary? {
+    if (!booking.isCancelled) {
       return bookingSummaryTransformer.transformJpaToApi(booking)
     }
-
     return null
   }
+
+  private fun getSpaceBookingSummary(bookings: List<Cas1SpaceBookingEntity>): BookingSummary? {
+    val bookings = bookings.filter { booking -> !booking.isCancelled() }
+    if (bookings.isEmpty()) return null
+    return cas1SpaceBookingSummaryTransformer.transformJpaToApi(bookings.first())
+  }
```

Comment:
i wonder if it would be more readable doing something like

```
bookings.firstOrNull { !it.isCancelled() }?.map { cas1SpaceBookingSummaryTransformer.transformJpaToApi(it) } 
```